app/packages/registration-schema/validations.ts

```diff
@@ -0,0 +1,98 @@
+import { assert } from '@ember/debug';
+import { ValidationObject, ValidatorFunction } from 'ember-changeset-validations';
+import { validateLength, validatePresence } from 'ember-changeset-validations/validators';
+import translations from 'ember-osf-web/locales/en/translations';
+import { PageResponse } from 'ember-osf-web/packages/registration-schema';
+import { SchemaBlockGroup } from 'ember-osf-web/packages/registration-schema/schema-block-group';
+import { validateResponseFormat } from 'ember-osf-web/validators/validate-response-format';
+
+export function buildValidation(groups: SchemaBlockGroup[]) {
+    const ret: ValidationObject<PageResponse> = {};
+    groups.forEach((group: SchemaBlockGroup) => {
+        let validationForResponse: ValidatorFunction[] = [];
+        const responseKey = group.registrationResponseKey;
+        assert(`no response key for group ${group.schemaBlockGroupKey}`,
+            responseKey !== '' || responseKey !== undefined || responseKey !== null);
```

Comment:
Each of these values is `falsy`, is there any reasons you didn't go with ` Boolean(responseKey)`?

Reply:
I did not realize that was a thing I could do! Makes it much more readable